bin/transform-collection.js

```diff
@@ -20,12 +20,17 @@ program
     .action(function (options) {
         transformer.convert(options, function (error, result) {
             if (error) {
-                console.log(error);
+                console.error(error.message);
```

Comment:
Some error might not have `.message`. A good precaution would be `error.message || error`.